src/App.jsx

```diff
@@ -14,33 +44,69 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const visibleGoods = getPreparedGoods(goodsFromServer, {
+    sortField,
+    isReversed,
+  });
+  const reset = () => {
+    setSortField('');
+    setIsReversed(false);
+  };
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={cn('button is-info', {
+            'is-light': sortField !== SORT_FIELD_ALPHABET,
```

Comment:
The condition `sortField !== SORT_FIELD_ALPHABET` might be clearer if you explicitly check for equality with `SORT_FIELD_ALPHABET` and use a ternary operator for setting the class.